default.nix

```diff
@@ -0,0 +1,41 @@
+{ pkgs? import <nixpkgs> {}
+, golem-examples ? pkgs.fetchFromGitHub { 
+    owner = "golemcloud";
+    repo = "golem-examples";
+    rev = "5165dcc7e3cbfa09f752caa96a869d284ec169aa";
+    hash = "sha256-7uWvvrRpIo4euFgP0TG4PEXENl4+Wgd8ckPpYnAwQbw=";
+  }
+}:
+pkgs.rustPlatform.buildRustPackage {
+  pname = "golem-cli";
+  version = "0.0.98";
+
+  src = pkgs.lib.cleanSource ./.;
+
+  cargoLock = {
+    lockFileContents = builtins.readFile ./Cargo.lock.nix;
+    allowBuiltinFetchGit = false;
+     outputHashes = {
```

Comment:
What is special about these two dependencies? What if they need to be updated? Or how do we know we need them or need something else here?

Reply:
not 100% sure why nix could not include this dependencies directly from lock file, I had to include them explicitly to resolve some conflicts.
in order to update them it could be done manually if they are not updated that frequently otherwise it is possible to add another command to update-deps command in commands.nix (dev shell)